scripts/Inelastic/CrystalField/fitting.py

```diff
@@ -498,6 +498,8 @@ def FWHM(self, value):
             if len(value) != len(self.Temperature):
                 if self.PhysicalProperty is not None and len(value) == len(self.Temperature) - len(self.PhysicalProperty):
                     value = value + [0] * len(self.PhysicalProperty)
+                    # Cast all types to match the first elem so we don't have mixed lists of int/doubles
+                    value = [type(value[0])(v) for v in value]
```

Comment:
I was just trying to maintain backwards compat :sweat_smile: since sometimes they passed a list of `int` values so I didn't want to break it

You're completely right about it making no sense as the fitting layer will turn it into a double ultimately